docs/content/how-to/kubevirt/ingress-and-dns.md

```diff
@@ -0,0 +1,213 @@
+# Ingress and DNS configuration
+
+By default, the HyperShift operator will configure the KubeVirt platform guest
+cluster's ingress and DNS behavior to reuse what is provided by the underlying
+infra cluster that the KubeVirt VMs are running on. This section describes
+that default behavior in greater detail as well as information on advanced usage
+options.
+
+## Default Ingress and DNS Behavior
+
+Every OpenShift cluster comes setup with a default application ingress
+controller which is expected have an wildcard DNS record associated with it.
+By default, guest clusters created using the Hypershift KubeVirt provider
+will automatically become a subdomain of the underlying OCP cluster that
+the KubeVirt VMs run on.
+
+For example, if an OCP cluster cluster has a default ingress DNS entry of
+`*.apps.mgmt-cluster.example.com`, then the default ingress of a KubeVirt
+guest cluster named `guest` running on that underlying OCP cluster will
+be `*.apps.guest.apps.mgmt-cluster.example.com`.
+
+!!! note
+
+    For this default ingress DNS to work properly, the underlying cluster
+    hosting the KubeVirt VMs must allow wildcard DNS routes. This can be
+    configured using the following cli command. ```oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'```
```

Comment:
Is there a way to make the patch command more visible here? It seems like ``` doesn't like `!!!note`

Reply:
it seems to display correctly when the docs are published, here's an example of how the note is interpreted today. https://hypershift-docs.netlify.app/how-to/kubevirt/create-kubevirt-cluster/#default-ingress-and-dns-behavior